code/src/components/Adjective.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const hornAdjectives = [
+  'golden',
+  'sparkling',
+  'shiny',
+  'colorful'
+]
+
+export const Adjective = ({ horn, setHorn }) => {
```

Comment:
This would be easier to connect to what it does with a name with 'horn' in it, when surveying your code, even though it makes sense.